src/Chirp.CLI/UserInterface.cs

```diff
@@ -6,15 +6,15 @@ namespace Chirp.CLI;
 
 class UserInterface
 {
-    public static void PrintCheeps(IEnumerable<Program.Cheep> records)
+    public static void PrintCheeps(IEnumerable<Cheep> records)
```

Comment:
Please add documentation to this method

Reply:
Not relevant to this commit (is about UserInterface). Too late buddy